tests/package-add-new-titles-test.js

```diff
@@ -69,11 +69,11 @@ describeApplication('PackageShowAllowKbToAddTitles', () => {
     });
 
     it('sets the state of allow KB to add titles to false', () => {
-      expect(PackageShowPage.allowKbToAddTitles).to.be.false;
+      expect(PackageShowPage.hasAllowKbToAddTitles).to.be.false;
     });
 
-    it.still('does not display the allow KB to add titles toggle switch', () => {
-      expect(PackageShowPage.allowKbToAddTitlesToggle).to.not.exist;
+    it.always.skip('does not display the allow KB to add titles toggle switch', () => {
```

Comment:
After a rebase with the new `@bigtest/mocha` upgrade, you should be able to unskip these

Reply:
Alright, done. Let's see what happens.

Reply:
Well, what happened was a new test failure. I have no idea why it is failing though.

Reply:
I see... But on the bright side, it looks like it's reliably failing 😅